### PR TITLE
Out of England new page

### PR DESCRIPTION
**Pull request by beseven**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by beseven** [comment]
I'm trying to get an agreement from the team that this is the way we are handling out of England postcodes so I can close #152 / cc @neilbmclaughlin @ajthomascouk @ermlikeyeah

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Comment by neilbmclaughlin** [comment]
Styling issue on form input labels:

screen shot 2017-07-05 at 18 05 49

File: app/views/includes/search-form.nunjucks [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <fieldset>
           <legend class="sr-only">LEGEND</legend>
           <div class="form-group form-group__search">
-            <label class="form-label form-label--bold" for="search">{{ searchLabel }}</label>
+            <label class="form-label form-label--bolYou can also search by your postcode" for="search">{{ searchLabel }}</label>
```
Comment (neilbmclaughlin) [inline app/views/includes/search-form.nunjucks]: I suspect this is causing the label issue mentioned in the other comment.

**Comment by beseven** [comment]
I think the outcode work can be done when adding any changes required from the UR review.

File: app/views/includes/search-form.nunjucks (hunk 1 above)
Comment (beseven) [inline app/views/includes/search-form.nunjucks]: I thought I fixed it in the previous one, fixing now!

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-150.dev.beta.nhschoices.net)

**Pull request merged**